base_phone_rate/models/phone_rate.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+from odoo.addons import decimal_precision as dp
+
+
+class PhoneRate(models.Model):
+    _name = "phone.rate"
+
+    name = fields.Char("Name", required=True)
+    country_id = fields.Many2one('res.country', "Country", required=True)
+    state_id = fields.Many2one('res.country.state', "Country State")
+    dial_prefix = fields.Char("Dial Prefix", required=True)
+    rate = fields.Float("Rate", required=True,
+                        digits=dp.get_precision('Phone Rate'))
```

Comment:
Idea for future/roadmap (would be a change request): 
I wonder if we should also include a "Product" here.
We might want to be able to set also for SMS, MMS, etc.